app/views/user_mailer/added_to_collection_notification.text.erb

```diff
@@ -1,13 +1,13 @@
 <% content_for :message do %>
-Dear <%= @user.default_pseud.byline %>,
+<%= t(".greeting", name: @user.default_pseud.byline) %>
 
-The collection maintainers of <%= @collection.title %> have added your work (<%= @work.title.html_safe %>) to their collection!
+<%= t(".part1", collection: @collection.title, work: @work.title.html_safe ) %>
```

Comment:
I think there's an extra space before the `)` here.

Reply:
Fixed